draft r-univ docs launch (#976)
ropensci/roweb3: content/blog/2025-02-25-r-universe-docs/index.md

We established a central place where we collect the various sources of information and describe examples and use cases, using popular authoring tools to support collective maintenance.

## New r-universe docs

The new R-universe documentation website is now live at <https://docs.r-universe.org>.
You can find the source code on [GitHub](https://github.com/r-universe-org/docs).

We created it as a [Quarto book](https://quarto.org/docs/books/) as this is a common format that many members of the R community are used to reading (for instance in [R Packages](https://r-pkgs.org/), [R for Data Science](https://r4ds.hadley.nz/), [our development guide](https://devguide.ropensci.org/)[^babelquarto]) and can contribute to.

The documentation is currently organized into four main sections:

- **Browse** that documents the user experience navigating packages and universes.
- **Install** that explains how to install packages from R-universe, how to ensure reproducibility, and how to create a dependency on a package published on R-universe.
- **Publish** that focuses on the package developer experience: why and how to publish packages.

> - **Infrastructure**, a very much in progress section aiming at documenting governance and funding but also how R-universe works under the hood.

Reviewer: ```suggestion
- **Infrastructure**, documenting governance, funding, and how R-universe works under the hood for those who want to contribute. (A work in progress!)
```

In addition the website has search functionality, and features links to the GitHub repository where you can contribute to the documentation.

![screenshot of r-universe documentation website](docs.png)

[^babelquarto]: Our dev guide is a special Quarto book, as it has multilingual support thanks to our R package [babelquarto](https://docs.ropensci.org/babelquarto).

We are thankful to the documentation reviewers [Lluís Revilla](/author/lluís-revilla-sancho/) and [Yaoxiang Li](/author/yaoxiang-li/) who provided insightful feedback that helped us improve the initial version of the website.

## Interactive website tour

Besides a new docs site, we also added some *embedded* documentation into the website: each package page has a ["Website tour" button](https://docs.r-universe.dev/browse/packages.html#website-tour) in the navbar, that guides you through all the information displayed on the page.
Go give it a try![^mobile]

[![screenshot of r-universe dashboard](textreuse.png)](https://ropensci.r-universe.dev/textreuse)

[^mobile]: The tour is not available on mobile.

## A streamlined docs experience

While working on the documentation we also updated other information sources:

> We simplified the [R-universe project page on the rOpenSci website](/r-universe) for it to only contain a big picture description of the project, as well as links to relevant resources including the documentation website, technical notes, recent talk recordings.

Reviewer: ```suggestion
We simplified the [R-universe project page on the rOpenSci website](/r-universe) the be easier to maintain. It now contains a big-picture description of the project, as well as links to relevant resources including the documentation website, technical notes, recent talk recordings.
```
